src/avatar.cpp

```diff
@@ -1245,6 +1245,21 @@ bool avatar::cant_see( const tripoint &p )
     return aim_cache[p.x][p.y];
 }
 
+bool avatar::cant_see( const tripoint_bub_ms &p )
```

Comment:
As always, you're doing great work here!

But, don't duplicate the function body in the typed version - just make the typed/untyped version called the untyped/typed version.